src/helper/shallowEqualSelector.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule shallowEqualSelector
+ */
+
+'use strict'
+
+import { createSelectorCreator, defaultMemoize } from 'reselect';
+import shallowEqual from 'shallowEqual';
+
+export default createSelectorCreator(defaultMemoize, shallowEqual);
```

Comment:
using this so our selectors do a shallowEqual check instead strict equals.  This allows organize our state into sub objects which reduces the number of params to selectors.